crossbeam-epoch/src/internal.rs

```diff
@@ -55,7 +55,7 @@ use crate::sync::queue::Queue;
 
 /// Maximum number of objects a bag can contain.
 #[cfg(not(feature = "sanitize"))]
-const MAX_OBJECTS: usize = 64;
+const MAX_OBJECTS: usize = 62;
```

Comment:
fwiw, if make `Local` to be less than or equal to 1024 bytes, `MAX_OBJECTS` needs to be 30 (1016 bytes)

Reply:
If I didn't misunderstand @stjepang's [this comment](https://github.com/crossbeam-rs/crossbeam/issues/551#issuecomment-684515013), It seems fine with less than half of what it is now, so it may be okay with this.